tests/sdk/nodejs/server-side-apply/step3/index.ts

```diff
@@ -0,0 +1,181 @@
+// Copyright 2016-2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as k8s from "@pulumi/kubernetes";
+
+// This test creates a Provider with `enableServerSideApply` enabled. The following scenarios are tested:
```

Comment:
Thank you for the comments here! I wish we did this in all the tests that are otherwise not exactly readable as to their intent.